src/karts/controller/skidding_ai.cpp

```diff
@@ -1132,24 +1140,36 @@ void SkiddingAI::evaluateItems(const Item *item, Vec3 kart_aim_direction,
 
 //-----------------------------------------------------------------------------
 /** Handle all items depending on the chosen strategy.
- *  Either (low level AI) just use an item after 10 seconds, or do a much
- *  better job on higher level AI - e.g. aiming at karts ahead/behind, wait an
- *  appropriate time before using multiple items etc.
+ *  Level 0 "AI" : do nothing (not used by default)
+ *  Level 1 "AI" : use items after a random time
+ *  Level 2 AI : strategy detailed before each item
```

Comment:
Maybe tweak the comments slightly to make it clearer that higher numbers = better AI

Reply:
Done